src/libstd/collections/hash/set.rs

```diff
@@ -292,21 +292,21 @@ impl<T, S, H> HashSet<T, S>
     ///
     /// ```
     /// use std::collections::HashSet;
-    /// let a: HashSet<int> = [1, 2, 3].iter().map(|&x| x).collect();
-    /// let b: HashSet<int> = [4, 2, 3, 4].iter().map(|&x| x).collect();
+    /// let a: HashSet<int> = [1, 2, 3].iter().cloned().collect();
+    /// let b: HashSet<int> = [4, 2, 3, 4].iter().cloned().collect();
```

Comment:
Ah, filtered out `src/libcollections/*` but this is in std, I'll do a quick rebase on master since that one's merged now.